app/utils/local_drive_identity.py

```
@@ -18,7 +18,7 @@ def _diskutil_info(path: Path) -> Dict:
         return {}
     try:
         proc = subprocess.run(
-            ["diskutil", "info", "-plist", str(path)],
+            ["diskutil", "info", "-plist", str(path.absolute())],
```

Ruff (0.15.11)

[error] Starting a process with a partial executable path

(S607)